src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/InvitationRepository.cs

```diff
@@ -139,12 +139,14 @@ public void UpdateRfocStatuses(string projectName, IList<string> commPkgNos, ILi
         {
             var project = _context.Projects.SingleOrDefault(x => x.Name.Equals(projectName));
 
+
             var invitations = _context.Invitations
                 .Include(i => i.McPkgs)
                 .Include(i => i.CommPkgs)
+                .ToList()
                 .Where(i => i.ProjectId == project.Id
                             && (i.CommPkgs.Any(x => commPkgNos.Any(y => y == x.CommPkgNo))
-                            || i.McPkgs.Any(x => mcPkgs.Any(y => y.Item1 == x.McPkgNo && y.Item2 == x.CommPkgNo))))
+                                || i.McPkgs.Any(x => mcPkgs.Any(y => y.Item1 == x.McPkgNo && y.Item2 == x.CommPkgNo))))
                 .ToList();
 
             foreach (var invitation in invitations)
```

Comment:
Will this work for the query further down?  UpdateRfocStatusForDP

Reply:
no